### ISSUE TITLE
No Spin logs in release

### ISSUE THREAD

**Issue by 0xE282B0** [deislabs/containerd-wasm-shims#186]
Hi,
I noticed that with the release (at least 0.9.3 and 0.10.0) there are again no logs from within the module.
When I check out the 0.10.0 tag and do a `make install` the logs are working.

Expected output with the `ghcr.io/deislabs/containerd-wasm-shims/example
s/spin-rust-hello:latest` image after calling `localhost:80/hello`:
```
Serving http://0.0.0.0:80
Available Routes:
  hello: http://0.0.0.0:80/hello
  go-hello: http://0.0.0.0:80/go-hello
Hello, world! You should see me in pod logs
```

Could it be something related to the binary stripping during release process?

**Comment by Mossaka** [comment]
> When I check out the 0.10.0 tag and do a make install the logs are working.

That't super strange. Umm I am not sure the binary stripping would affect logging as it's set up with spin's hooks.

**Comment by 0xE282B0** [comment]
Hi,
There was indeed no problem with the release, only the log message was removed from the test image in [this commit](https://github.com/deislabs/containerd-wasm-shims/commit/e5e2679136fd66f022662a5efb97df0fb841fc54#diff-5e111c8d53153d69c210c55fadfb4a0edd463cfd5a4fbde1abd1aec086cc3691L8-R7) a week before. I guess I had an old version cached in my cluster where the log message was still there, and with the K3d quickstart cluster I didn't see it because it was removed from the code.

In short, the actual behavior of the test image is that the `/hello` endpoint does not log anything, but the `/go-hello` endpoint does. I'll close this issue.
